twentytwentyone-blocks/assets/css/style-shared.css

```diff
@@ -8,16 +8,6 @@ body {
 	-webkit-font-smoothing: antialiased;
 }
 
-h1,
-.has-huge-font-size,
-.has-gigantic-font-size {
-	font-weight: var(--wp--custom--font-weight-light);
-}
```

Comment:
This was kind of a weird hack the theme did — we wanted to make sure that the largest font size presets all used a light weight. Not the best solution. 😅 

It's fine to remove this, but we should try to make sure the `h1` begins using the light font weight preset [once it's available to that block](https://developer.wordpress.org/block-editor/developers/themes/theme-json/#typography-properties).